plugins/module_utils/checkpoint.py

```diff
@@ -1680,9 +1680,9 @@ def api_call(
             auto_publish_session = payload["auto_publish_session"]
             del payload["auto_publish_session"]
         if state == "merged":
-            if equals_response.get("equals") and not equals_response.get(
+            if equals_response.get(
                 "equals"
-            ):
+            ) == False and not equals_response.get("equals"):
```

Comment:
Isn't "equals_response.get("equals")==False" the same as "not equals_response.get("equals")"

Reply:
true, but I wanted to eliminate the `None` scenario where for `not equals_response.get("equals")` will result into pass scenario. I've updated the order to make more sense.

Reply:
"equals_response.get("equals")==False" should eliminate both 'True' & 'None' scenario

Reply:
@chkp-edenbr ah my bad, I've updated the logic to eliminate the above-said scenario, else for `None` value it'll get `AttributeError: 'NoneType' object has no attribute 'get'`